src/interfaces.ts

```diff
@@ -163,6 +163,13 @@ export interface ExtismPluginOptions {
   functions?: { [key: string]: { [key: string]: (callContext: CallContext, ...args: any[]) => any } } | undefined;
   allowedPaths?: { [key: string]: string } | undefined;
   allowedHosts?: string[] | undefined;
+
+  /**
+   * Whether WASI stdout should be forwarded to the host.
+   *
+   * Overrides the `EXTISM_ENABLE_WASI_OUTPUT` environment variable.
+   */
+  enableWasiOutput?: boolean | undefined;
```

Comment:
just curious, are the `?` suffixes necessary when the type definition is `x | undefined`?

Reply:
Nope! Good catch, thanks! – removed those.